File: Python Excel Examples/WorksheetsApiDemo/named_ranges/getNamedRangeValue.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "GetNamedRangeValue.xlsx"
storage = ""
folder = "ExcelDocument"
namerange = "Name1"
result=api.get_named_range_value(name, namerange, folder=folder, storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/NamedRanges/getNamedRangeValue.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "GetNamedRangeValue_1.xlsx";
$namerange = "Name1";
$folder = "input";
$storage = null;
$result = $apiInstance->getNamedRangeValue($name,$namerange, $folder, $storage);